src/BaseForm/BaseForm.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { BaseForm, FormObject } from './BaseForm';
+
+export default {
+    title: 'BaseForm',
+    component: BaseForm,
+} as ComponentMeta<typeof BaseForm>;
+
+type FormFields =
+  | 'firstName'
+  | 'lastName'
+  | 'optInEmail'
+  | 'rootCanal'
+  | 'eddieDingle';
+
+export const UncontrolledInputs: ComponentStory<typeof BaseForm> = () => {
+    const [formData, setFormData] = useState<FormObject<FormFields>>();
+    return (
+        <BaseForm<FormFields>
+            title="Uncontrolled Inputs"
+            onSubmit={(data) => setFormData(data)}
+        >
+            <h3>All inputs are uncontrolled</h3>
+            <input name="firstName" placeholder="first name" required />
+            <input name="lastName" placeholder="last name" required />
+            <br />
+
+            <label>
+        I would like to receive emails <br />
+                <input type="checkbox" value="true" name="optInEmail" required />
+            </label>
+            <br />
+
+            <label>
+        I would like to get a root canal <br />
+                <input type="checkbox" value="true" name="rootCanal" required />
```

Comment:
This insane formatting is brought to you courtesy of prettier. I don't care much because it's a story, but need to revisit the prettier config at some point.